website/source/docs/providers/aws/r/config_config_rule.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "aws"
+page_title: "AWS: aws_config_config_rule"
+sidebar_current: "docs-aws-resource-config-config-rule"
+description: |-
+  Provides an AWS Config Rule.
+---
+
+# aws\_config\_config\_rule
+
+Provides an AWS Config Rule.
+
+~> **Note:** Config Rule requires an existing [Configuration Recorder](/docs/providers/aws/r/config_configuration_recorder.html) to be present. Use of `depends_on` is recommended (as shown below) to avoid race conditions.
```

Comment:
Is `depends_on` really necessary here? Wouldn't the use of an interpolation make sure the graph orders the creation?